components/profileForm.tsx

```diff
@@ -136,34 +153,42 @@ function ImageUpload({
   return (
     <>
       {!image ? (
-        <label
-          className="mb-5"
-          style={{
-            border: 'white 3px dotted',
-            display: 'flex',
-            borderRadius: 12,
-            justifyContent: 'center',
-            alignItems: 'center',
-            overflow: 'hidden',
-            cursor: 'pointer',
-          }}
-        >
-          <span className="after:ml-0.5 after:text-red-500 after:content-['*']">
-            画像をアップロードする
-          </span>
-          <input
-            className="h-1 w-1 opacity-0"
-            type="file"
-            accept="image/*"
-            id={imageId}
-            {...register('image', { required: true })}
-            alt="Upload Image"
-            onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
-              setImage(e.target.files)
-            }
-            required={true}
-          />
-        </label>
+        <>
+          <label className="block font-medium text-gray-900 text-sm leading-6">
+            <span className="after:ml-0.5 after:text-red-500 after:content-['*']">
+              障がい者手帳の画像・写真
+            </span>
+          </label>
+          <div className="mt-2 flex justify-center rounded-lg border border-gray-900/25 border-dashed px-6 py-10">
+            <div className="text-center">
+              <PhotoIcon
+                className="mx-auto h-12 w-12 text-gray-300"
+                aria-hidden="true"
+              />
+              <div className="mt-4 flex text-gray-600 text-sm leading-6">
+                <label className="relative cursor-pointer rounded-md bg-white font-semibold text-indigo-600 hover:text-indigo-500 focus-within:outline-none focus-within:ring-2 focus-within:ring-indigo-600 focus-within:ring-offset-2">
+                  <span>アップロード</span>
+                  <input
+                    type="file"
+                    className="sr-only"
+                    accept="image/*"
+                    id={imageId}
+                    {...register('image', { required: true })}
+                    alt="Upload Image"
+                    onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                      setImage(e.target.files)
+                    }
+                    required={true}
+                  />
+                </label>
+                <p className="pl-1">又は、ドラッグ＆ドロップ</p>
+              </div>
+              <p className="text-gray-600 text-xs leading-5">
+                PNG, JPG, GIF, WEBP のファイルを 100MB まで
+              </p>
```

Comment:
I tried to drag & drop a file smaller than 100MB, but it seems not to work well and not to be able to preview the image.
Have you tried and it has worked well?

Then, because I don't have so big image file, I haven't upload a file larger than 100MB. Have you checked the case?

Reply:
This is merely appearance.
I haven't implemented the feature of upload limit.

I will add upload limitation function in the future.

Reply:
IMO at first, make frontend area.
Second, the feature related to the frontend which was made.

Then do not worry about that functions.
But thank you for your nice review!